### ISSUE TITLE
Card view

### ISSUE THREAD

**Issue by jayabie** [Phuks-co/throat#515]
Is the card view like https://poal.co/ implemented in throat by default or needs a fork modification?

**Comment by Polsaker** [comment]
Poal is a closed-source fork and most of their changes haven't been submitted to us

**Comment by jayabie** [comment]
@Polsaker Yes i understand, my question was, if i install throat from the master branch, does it come with card view theme or i have to fork it and modify it?

**Comment by Polsaker** [comment]
Throat does not have that feature yet, so you'll have to add it yourself, yeah.
